.github/workflows/ci.yml

```diff
@@ -41,6 +41,8 @@ jobs:
         run: make dependabot-check license-check lint vanity-import-check
       - name: Build
         run: make build
+      - name: Run code generators
+        run: make crosslink
```

Comment:
How about also adding it here https://github.com/open-telemetry/opentelemetry-go/blob/main/Makefile#L29?

Side note: Any reason why the GH workflow is not simply executing `make ci`?

Reply:
It seems `make ci` sets things up and runs the tests, when this part of the workflow only sets things up, and they later run in separate steps?